contrib/deploy-onefuzz-via-azure-devops/deploy-onefuzz.yml

```diff
@@ -69,7 +69,7 @@ stages:
             workingDirectory: "$(ONEFUZZ_DEPLOY_LOC)/$(artifact)"
             script: |
               set -ex
-              unzip onefuzz-deployment-$(onefuzz_release.version).zip
+              unzip -DD onefuzz-deployment-$(onefuzz_release.version).zip
```

Comment:
IIUC, the issue is that (1) the new files _were_ getting correctly extracted on the build machine, but (2) indirect invocations of `azcopy sync` were not remotely updating some genuinely new files, because it takes an action/no-action based on local vs. remote timestamps.

If that's true, does that mean we should consider changing [the `azcopy` invocation](https://github.com/microsoft/onefuzz/blob/main/src/deployment/deploy.py#L579), e.g. to `azcopy copy`?

Reply:
I believe either solution would resolve the issue.

Reply:
We should use `azcopy copy`, as `unzip -DD` isn't portable.

Reply:
There are two different places in deploy.py that invoke the 'sync' command and changing one and not the other could lead to confusion. Any objection to changing both calls? 

One is for 'tools' dir and one is for 'instance-specific-setup' dir.

Reply:
Both should be updated.

Reply:
Thanks Brian. I'm happy to go ahead and push the changes to deploy.py. Please let me know if you'd rather do it yourself, but otherwise, I will update.

Reply:
You're welcome to make the updates.

Reply:
There appears to be 3 places where azcopy "sync" is used. Should I make all 3 changes?

Reply:
Yes.